CMakeLists.txt

```diff
@@ -210,6 +217,7 @@ if(NOT WIN32)
         core_optional_dep(LibDRM ENABLE_MIR)
       else()
         core_optional_dep(X ENABLE_X11)
+        core_optional_dep(XRandR ENABLE_X11)
```

Comment:
I certainly won't merge this in this state.

1)
X11, AML, IMX are platforms, means they define their own windowing system. I don't see what you intend Rockchip to be. Without supporting any of the hw accelerators APIs we have avaialbe for X11, it makes zero sense running a kodi build for X11 on that processor because the CPU is not powerful enough.

2)
If rockchip intends to be a platform like AML, it has to provide its own implementeations for the egl windowing system. Assuming that the port for AML will always do is wrong.